src/celeritas/CMakeLists.txt

```diff
@@ -164,6 +164,19 @@ if(CELERITAS_USE_CUDA OR CELERITAS_USE_HIP)
   )
 endif()
 
+if(CELERITAS_USE_Covfie)
+  set(_covfie_backend covfie::cpu)
+  if(CELERITAS_USE_CUDA)
+    set(_covfie_backend covfie::cuda)
+  elseif(CELERITAS_USE_HIP)
+    set(_covfie_backend covfie::hip)
+  endif()
+  list(APPEND PUBLIC_DEPS covfie::core ${_covfie_backend})
+  list(APPEND SOURCES
+    field/CartMapFieldParams.cc)
+  celeritas_polysource(alongstep/AlongStepCartMapFieldMscAction)
```

Comment:
The along-step action should (after we hide dependencies) be built regardless of Covfie being enabled. The `CartMapFieldParams.cc` can have our `#if !CELERITAS_USE_COVFIE ... (not implemented)` logic.